src/theme/index.css

```diff
@@ -212,6 +214,21 @@ body {
   background: var(--primary-color) !important;
 }
 
+.fvofykd {
+  background: #1a1a1a !important;
+}
+.wrapper_f1mi6c8p {
+  background: rgb(26, 26, 26) !important;
+}
+
+.f1q0y4o5 {
+  color: white !important;
+}
+
+.fofzirn {
+  background: #1a1a1a !important;
+}
+
```

Comment:
We already support these colors - see `:root` - please use `var(--primary-color)`, or `var(--text-color)`.